src/components/Icon/Icon.scss

```diff
@@ -216,3 +216,7 @@ $stacking-order: (
   padding-bottom: 100%;
   background: currentColor;
 }
+
+.colorDarkGrey {
+  fill: color(darkgrey);
```

Comment:
@ankitsaraogi or @SadafAdrees this is not a value color value. Please select a valid mapped value from https://github.com/emgage/engage-ui/blob/master/src/styles/foundation/colors.scss . Please fix this ASAP as it's breaking the engage-ui build ( https://dev.azure.com/ateraplat/Emgage%20Platform/_build?definitionId=34 )

Reply:
@ankitsaraogi or @SadafAdrees now that I look at this again, why is 'darkGrey' Color prop even being added? You can simply use the existing "inkLight" prop which will give the icon the #637381 color value. Please simply remove this new color prop value. Thanks

Reply:
@weegrom Ok, Thanks!

Reply:
@weegrom  Can you please approve this PR https://github.com/emgage/engage-ui/pull/615